packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -2839,6 +2839,114 @@ void main() {
       platform: globals.platform,
     ),
   });
+
+  testUsingContext('CADisableMinimumFrameDurationOnPhone is true in info.plist for objc iOS project.', () async {
```

Comment:
Instead of adding new tests for this plist value, what if you change the `UIViewControllerBasedStatusBarAppearance` tests to check expected Info.plist values.

https://github.com/flutter/flutter/blob/570e39d38b799e91abe4f73f120ce494049c4ff0/packages/flutter_tools/test/commands.shard/permeable/create_test.dart#L1380-L1381
```dart
    final bool viewControllerBasedStatusBarAppearance = _getBooleanValueFromPlist(plistFile: plistFile, key: 'UIViewControllerBasedStatusBarAppearance');
    expect(viewControllerBasedStatusBarAppearance, true);
    final bool disabled = _getBooleanValueFromPlist(plistFile: plistFile, key: 'CADisableMinimumFrameDurationOnPhone');
```